storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -1872,6 +1872,13 @@
     @isString
     public static final String TOPOLOGY_BOLTS_TUPLE_TIMESTAMP_FIELD_NAME = "topology.bolts.tuple.timestamp.field.name";
 
+    /**
+     * Bolt-specific configuration for windowed bolts to specify whether late tuples should be emitted on a stream
+     * called _late, instead of being logged with INFO level.
+     */
+    @isBoolean
+    public static final String TOPOLOGY_BOLTS_EMIT_LATE_TUPLE = "topology.bolts.emit.late.tuple";
```

Comment:
Yes it sounds reasonable. An exception can be thrown if the `lateTupleStream` is set in the config but the same is not defined in the bolt's output streams.